Client/WarFare/UIVarious.cpp

```diff
@@ -699,6 +704,34 @@ void CUIKnights::RefreshButtonHandler(bool blBypassTime)
 	MsgSend_MemberInfoAll();
 }
 
+void CUIKnights::ClanPartyButtonHandler()
+{
+	int iSel = m_pList_CharIDs->GetCurSel();
+	if (iSel == -1)
```

Comment:
You can remove this check, and just check the result of `GetString()`.
It returns a bool depending on whether or not it's a valid index.